internal/hcs/vmcompute.go

```diff
@@ -292,6 +292,15 @@ func hcsGetServicePropertiesContext(ctx gcontext.Context, propertyQuery string,
 	})
 }
 
+func hcsGetServicePropertiesSync(ctx gcontext.Context, propertyQuery string, properties **uint16, result **uint16) (hr error) {
```

Comment:
>hcsGetServicePropertiesSync [](http://example.com/codeflow?start=5&length=27)

hcsGetServiceProperties should be sufficient

Reply:
fix